src/test/scala/com/crealytics/spark/excel/IntegrationSuite.scala

```diff
@@ -18,16 +18,16 @@ import org.apache.spark.sql.functions.lit
 object IntegrationSuite {
 
   case class ExampleData(
-    aBoolean: Boolean,
-    aByte: Byte,
-    aShort: Short,
-    anInt: Int,
-    aLong: Long,
-    aDouble: Double,
-    aString: String,
-    aTimestamp: java.sql.Timestamp,
-    aDate: java.sql.Date
-  )
+                          aBoolean: Boolean,
```

Comment:
This indentation style leads to unnecessary git diffs when `ExampleData` gets renamed to something shorter or longer. Would you be so kind to change it back?

I will add a scalafmt configuration in a few minutes, then all formatting questions should be automated :wink:

Reply:
I will fix it.. it didn't show up in my merge tool sorry